index.html

```diff
@@ -13461,6 +13453,83 @@ <h2>Including Elements in the Accessibility Tree</h2>
             </li>
           </ul>
 	</section>
+	<section id="tree_relationships">
+	  <h2>Relationships in the Accessibility Tree</h2>
+          <p>The following terms are used to describe relationships between <abbr title="Document Object Model">DOM</abbr> elements.</p>
+          <p>The <dfn data-export="" data-lt="accessibility child|owned child|child element|child|children|child elements">accessibility children</dfn> of a DOM element are all of the children of that element's corresponding <a>accessible object</a> in the <a>accessibility tree</a>. In terms of the DOM, that includes the following:</p>
+          <ul>
+            <li>The <abbr title="Document Object Model">DOM</abbr> children of the <a>element</a>.
```

Comment:
It might be worth adding "(exceptions below)" to the end of this sentence, for people who are skimming, but use your own judgement on whether that's worth it or not

Reply:
Ah yeah you might not see but I had the exceptions listed in a parenthetical above, but it was a bit too much, so I moved them below, but I think I'll incorp your idea!